src/rest/getManagerHandler_test.go

```diff
@@ -0,0 +1,25 @@
+package rest
+
+import (
+	"github.com/kataras/iris/v12/httptest"
+	"net/http"
+	"testing"
+)
+
+func TestGetManager(t *testing.T) {
```

Comment:
Please unify naming convention for tests - tests for basicAuthHandler are containing `_` while here we have camelCase. My proposition: `Test_*` where `*` is the name for a test - `Test_GetManager` instead of `TestGetManager`

Reply:
Refactored.

Reply:
`TestGetStatus` is still not updated

Reply:
Refactored.